pkg/config/containers.conf

```diff
@@ -340,6 +340,10 @@ default_sysctls = [
 #  "/usr/lib/netavark",
 #]
 
+# The default firewall driver to be used
+# firewall_driver = "iptables"
```

Comment:
```suggestion
# The firewall driver to be used by netavark.
# The default is empty which means netavark will pick one accordingly. Current supported 
# drivers are "iptables", "none" (no firewall rules will be created) and "firewalld" (firewalld is
# experimental at the moment and not recommend outside of testing). In the future we are
# planning to add support for a "nftables" driver.
#firewall_driver = ""
```
As text for the documentation. @mheon WDYT?

Reply:
Should mention that the default is iptables - maybe "Netavark will use the default (iptables)" instead of "netavark will pick one accordingly"

Reply:
Yeah I was thinking about it but then I remembered that we do not really control what nv version is on the system and if we decide to flip the default to nftables once it is ready I would assume that we forget to update the page here.

Reply:
Fair enough. We might also add some sort of autodetection for systems without iptables but with nft at some point as well, which would affect the default.